Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -69,14 +70,19 @@ public ExportDms(boolean exportImages) {
      *            Process object
      */
     @Override
-    public void startExport(Process process) throws IOException, DAOException {
+    public void startExport(Process process) throws DAOException, DataException, IOException {
```

Comment:
DAOException and IOExceotion are never thrown in this method. So if you are already changing the signatur, remove not needed exceptions.